test/Microsoft.AspNet.Server.KestrelTests/TestServer.cs

```diff
@@ -48,5 +53,10 @@ public void Dispose()
             _server.Dispose();
             _engine.Dispose();
         }
+
+        public static int GetNextPort()
+        {
+            return _nextPort++;
```

Comment:
If the tests are running in parallel, we might need to use `Interlocked.Increment` to avoid random failures.